go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/project-kessel/inventory-api
 
 go 1.22.9
+toolchain go1.24.1
```

Comment:
the toolchain directive will force the bulid process to pull down and use the defined go version, in this scenario go 1.24

We cannot leverage upstream go in order to comply with FIPS at this time.